### DISCUSSION TITLE
General disscusion/Offtopic KSP

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#12] [Collaborator Disscusion]
VERY_COOL_PLACEHOLDER

**Comment by UncleMateo** [comment]
![image](https://github.com/user-attachments/assets/6929be1a-95d4-4f10-bd21-7c7b26ff3ec1)

**Reply by likeproblem** [reply]
No not him again ARGH

**Comment by Spartwo** [comment]
Ought to bunch any model swaps into a pull request for ReStock itself at the end

**Reply by likeproblem** [reply]
May I ask what you did with Separator4? Also renamed mesh? There's one thing, please make sure textures are present, because silly me forgot to include them into shielded fix and will do it right now, I think it is in blender .mu plugin settings and setup shader

**Reply by Spartwo** [reply]
renamed mesh

**Reply by likeproblem** [reply]
I somehow messed up textures in my model, and I can't fix it, so I am gonna manually apply them using TU's MATERIAL in Additional.cfg

**Comment by UncleMateo** [comment]
Yo I just want to apologize- doesn't normally take me 3 days to do 2 parts, but these claws (klaws?) have been pernicious and quarrelsome, for lack of more profane and non-github appropriate language.. also I've just been busy and the total free time I've had in the last few days has been a bit less than normal.  I almost have both parts working, but even using the config maker I find ways to make stupid mistakes.
Anyways I just wanted to post an update, proof of life, that sort of thing.  Should (hopefully) have them tonight, but if I still have some issues, might be tomorrow instead.  I gotta go to work for a few hours, will hopefully post those claws soon.

**Reply by likeproblem** [reply]
I'm in the same boat, barely any free time, I feel you, it's hard to make progress with little amount of time

If you can't fix problems with configs, just ask for help

**Comment by likeproblem** [comment]
@Spartwo @UncleMateo The science tab is fully complete(though I think robot arms might need recolour)!

**Comment by likeproblem** [comment]
@Spartwo @UncleMateo @EmberSkyMedia forums are back and running! :D

**Reply by Spartwo** [reply]
Was just about to tell you, chuck up them screenshots

**Reply by likeproblem** [reply]
When I get time I will post something on forum thread

**Reply by UncleMateo** [reply]
After work I'll plop some up there, while I'm finishing putting pictures up on the front page of the wiki.
just takes a couple minutes to set them all up for a group photo.. and for those with lots of parts, might have to put multiple screenshots.. while I'm doing those more formalized ones, might do maybe a giant sweeping shot like I did with PlanetaryBaseInc here: 
![image](https://github.com/user-attachments/assets/9dad7855-fb47-492c-b8fe-cdc431d1e838)
Zoomed out, minus the infrastructure, for the forums, of everything we've got so far
I think it could be quite impressive, but it would take a couple minutes even to set up.
Just an idea.

**Reply by Spartwo** [reply]
I prefer displays seperated by category. Advantages being as they're finished you don't need to update the screenshots and you can make out better detail in each.

**Reply by UncleMateo** [reply]
I think that makes a lot of sense, the more modular the better.

**Comment by Spartwo** [comment]
@UncleMateo 

https://github.com/Spartwo/KP-Modpack/tree/main/Recommended%20GameData/TURD/TU_APP_Recolour

I have a ramp config dumped in here if you're picking up cold war aero. The folder is a combiation APP/CWA/NMB

**Reply by UncleMateo** [reply]
I will gladly use it! Thanks!

**Comment by likeproblem** [comment]
Since we are getting closer to 50% mark, aka 170 parts, just wanted to ask you. I have several ideas for that: first off, proper release on Spacedock+CKAN, with removed .afphot and .pdn files. Second, make proper album. Third, of course announce everywhere.

Also one idea about album. When we add new parts, instead of redoing the photo, we should just add them nearby.

**Reply by likeproblem** [reply]
And obviously lazy painter

**Reply by EmberSkyMedia** [reply]
I can help with Album & Announcements (YouTuber duh). Not sure how to do space dock but I can find out (I need to do it for my other mods I want to release).

**Reply by likeproblem** [reply]
Spacedock is on me

About album and announcements, that would be so good! About announcements, I wanted to post on TURD and our thread, and on Reddit

**Reply by UncleMateo** [reply]
Very exciting!

Once we get it on CKAN, is the plan to have a dev branch? Or are we just going to do updates all the time?

**Reply by likeproblem** [reply]
We will just post releases on milestones, like 75%, after full release just bugfixes and quality check

**Comment by likeproblem** [comment]
@UncleMateo would you need help with screenshots? I'm more than happy to take them(I see you love no hangar mod)

**Reply by likeproblem** [reply]
btw you forgot to activate main switch in communication pic with @EmberSkyMedia 's antennas, I see only foil active

**Reply by UncleMateo** [reply]
Dang it!  Sorry my bad.

Yes, I would love if you helped with screenshots.  I just really wanted to get something up there, I know from my own experience that being able to see screenshots is the best way to **GET HYPE**

**Comment by likeproblem** [comment]
@Spartwo @EmberSkyMedia @UncleMateo publishing pre_release to Spacedock to avoid waiting apporval to ckan before actual 50% release, please send your profiles so I can add them as authors

**Reply by EmberSkyMedia** [reply]
I’m assuming you mean this…
https://spacedock.info/profile/PicoSpaceIndustries

**Reply by likeproblem** [reply]
Pending

**Reply by UncleMateo** [reply]
https://spacedock.info/profile/UncleMateo

**Reply by Spartwo** [reply]
I'll make one

**Reply by Spartwo** [reply]
Nevermind I have one under this name

**Reply by likeproblem** [reply]
You both have exact name to name profiles? Okay check your email then

**Reply by Spartwo** [reply]
> You both have exact name to name profiles? Okay check your email then

Is this why I couldn't find a discord account

**Reply by likeproblem** [reply]
Do you mean my discord account? Do not ask origins of my github profile lol

Also we are somehow getting downloads out of thin air

**Reply by Spartwo** [reply]
I've been telling people about the repo while the forum was down, KP, halban server, ksp modding community, etc. So people are excited. Also the spacedock reload counter isn't individuals it's total downloads. You can download the same thing more than once to drive up the value

**Reply by UncleMateo** [reply]
Pretty exciting seeing that download counter go up

**Comment by likeproblem** [comment]
I'm gonna work on leftover 3 parts and prepare for release, then release+announce and wait for CKAN

**Reply by likeproblem** [reply]
Overall, this is a huge milestone! Thanks to all of you for working hard!

**Reply by UncleMateo** [reply]
Just put up a few parts, and it looks like we passed the **50%** threshold!

**Reply by likeproblem** [reply]
Aww man, welp I will submit what I have, and we will work on release and albums instead

**Reply by likeproblem** [reply]
I sadly have barely any free time now

**Reply by UncleMateo** [reply]
Just plug in the time where you can, don't overburden yourself, and don't feel bad if you can't contribute to the project every single day.  We are in a marathon here, not a 5k.  If you gotta take a break, by all means, do it.

**Reply by Spartwo** [reply]
I still intend to complete out the parts I have associated

**Comment by UncleMateo** [comment]
Hey guys..
I was wondering if someone who has a reddit account could make a post emphasizing that we are in open development and they can use it even though it's incomplete, there just will be parts that don't recolor (yet), and so long as they don't mind that, they can use it.  I noticed a post or two that seemed not to understand that.

In fact I so like the idea of open development I might start doing it myself.  Before I finished all the parts in a mod before uploading it, rather than piecemeal, a few parts at a time.  I see nothing wrong with letting people be interested in the final product, and expressing that by being more-or-less a beta tester, more because there are chances for random glitches, and less because recolors can't really break anything that isn't purely aesthetic.

Anyways I don't have a reddit account because I kind of dislike that place overall and pretty much exclusively use it for lurking the KSP subreddit.  (Full disclosure, one of my things over the years was trying to reverse engineer designs from just seeing them.. I had an aversion to using other people's designs from both KerbalX and the stock craft files, so, to be quite honest, I've never used either at all.  Instead, if I saw a neat design I'd try and study it as much as possible, maybe make a little sketch, and see if I could replicate it.  I wasn't one to brag about my own designs so there was no reason to do anything but lurk.. and as an added bonus, it was a good way to hear about mods.)  You guys don't have to do anything if you don't want to but I know there's a lot of hype surrounding our project here, and there's probably people out there that want to play around with what is working so far, might be a relatively low effort way to raise RestockRecolour awareness, slap a screenshot or two up there and say something like "_hey guys didja know you can use this cool-ass mod even though it's not exactly done yet? we are just that dang cool, we thought we'd let you check it out._"

Anyways, just an idea.. I do notice we get a lot of mentions there.. but also that a lot of people that know about TURD and know about Restock are somehow out of the loop that we are actually working a project to join them in holy matrimony.  Perhaps I am alone in this, but I think we should continue our 'public relations campaign' to try and get more people engaged.

**Reply by UncleMateo** [reply]
Oh also we passed 100 downloads on SpaceDock!

**Reply by likeproblem** [reply]
I have posted already two posts about it on 25% and 50%, and some volunteers already share it on different posts on reddit
